From be16fc7e6249279a2244d7761002c3bab9caadfa Mon Sep 17 00:00:00 2001
From: Priyanka Bose <priyanka.bose@intel.com>
Date: Wed, 20 Feb 2019 15:49:09 -0500
Subject: [PATCH] [PATCH] Adding dependancies for Capstone Disassembler in
 Vendor Art

Change-Id: I280791e6904ff8b94d577c66c084fe1661652b45
Category: device enablement
Domain: AOSP.ART-Other
Origin: internal
Upstream-Candidate: no, proprietary
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-68865
Signed-off-by: Priyanka Bose <priyanka.bose@intel.com>
---
 android/config.go   |  4 ++++
 android/variable.go | 10 ++++++++++
 2 files changed, 14 insertions(+)

diff --git a/android/config.go b/android/config.go
index fafed6b0..794241a7 100644
--- a/android/config.go
+++ b/android/config.go
@@ -845,3 +845,7 @@ func stringSlice(s *[]string) []string {
 		return nil
 	}
 }
+
+func (c *config) AutoFastJni() bool {
+        return Bool(c.productVariables.AutoFastJni)
+}
diff --git a/android/variable.go b/android/variable.go
index 1039299f..4ff151fe 100644
--- a/android/variable.go
+++ b/android/variable.go
@@ -100,6 +100,14 @@ type variableProperties struct {
 			Enabled *bool `android:"arch_variant"`
 		} `android:"arch_variant"`
 
+
+		AutoFastJni struct {
+                        Srcs          []string
+                        Include_dirs  []string
+                        Static_libs   []string
+                        Shared_libs   []string
+                }
+
 		Uml struct {
 			Cppflags []string
 		}
@@ -235,6 +243,8 @@ type productVariables struct {
 	PgoAdditionalProfileDirs []string `json:",omitempty"`
 
 	VendorVars map[string]map[string]string `json:",omitempty"`
+	
+	AutoFastJni *bool `json:",omitempty"`
 
 	// include Bliss variables
 	Bliss android.ProductVariables
-- 
2.17.1

